content/en-us/reference/engine/classes/Stats.yaml

```diff
@@ -97,8 +97,10 @@ properties:
     capabilities: []
     writeCapabilities: []
   - name: Stats.FrameTime
-    summary: ''
-    description: ''
+    summary: |
+      A measurement of how long it takes for the engine to process all tasks required to render a frame.
+    description: |
+      FrameTime is a measurement of how long it takes for the engine to render a frame. This can be used to calculate frames per second (FPS) through dividing 1 by this value. A high frame time value indicates the engine and device struggling to render frames.
```

Comment:
```suggestion
      This property is a measurement of how long it takes for the engine to render a frame. It can be used to calculate frames per second (FPS) through dividing `1` by this value. A high frame time value indicates the engine and device are struggling to render frames.
```